packages/cli/src/zosfiles/create/vsam/vsam.handler.ts

```diff
@@ -30,7 +30,6 @@ export default class VsamHandler extends ZosFilesBaseHandler {
             dataclass: commandParameters.arguments.dataClass,
             retainFor: commandParameters.arguments.retainFor,
             retainTo: commandParameters.arguments.retainTo,
-            showAttributes: commandParameters.arguments.showAttributes || commandParameters.arguments.attributes,
```

Comment:
Please note that this `showAttributes` is different from the `commandParameters.argumetns.showAttributes`.
we will still want to pass in the `showAttributes: commandParameters.arguments.attributes` since that's what will print the allocation attributes when someone creates a dataset (VSAM or regular)

Reply:
Here is an example
```
zFernand0 @ SSH:~/gh/zowe/zowe-cli# zdev files create pds rijfe01.public.test01.test01 -a
Data set created successfully.

zFernand0 @ SSH:~/gh/zowe/zowe-cli# zowe files create pds rijfe01.public.test01.test02 -a
alcunit:   CYL
dsorg:     PO
primary:   1
dirblk:    5
recfm:     FB
blksize:   6160
lrecl:     80
secondary: 1

Data set created successfully.
```

Reply:
Please add back the following:
```
showAttributes: commandParameters.arguments.attributes,
```